src/controllers/user.controller.js

```diff
@@ -106,7 +135,7 @@ const loginUser = async (req, res, next) => {
    const userEml = req.body.email;
    const user = await findUserByEmail(userEml);
     if (user == false) {
-      res.status(404).json("User not found");
+      res.status(404).json('User not found');
```

Comment:
please send a good-formated response

ex:  res.status(404).json({message: 'User not found'})